plugin/core/sessions.py

```diff
@@ -1100,6 +1169,12 @@ def _handle_initialize_success(self, result: Any) -> None:
         code_action_kinds = self.get_capability('codeActionProvider.codeActionKinds')
         if code_action_kinds:
             debug('{}: supported code action kinds: {}'.format(self.config.name, code_action_kinds))
+        semantic_token_types = self.get_capability('semanticTokensProvider.legend.tokenTypes')
```

Comment:
Would be better to tell it the right type:
```suggestion
        semantic_token_modifiers = cast(List[str], self.get_capability('semanticTokensProvider.legend.tokenModifiers'))
```